conans/assets/templates/new_v2_cmake.py

```diff
@@ -140,6 +220,88 @@ def get_files(name, version, package_name="Pkg"):
              "test_package/conanfile.py": test_conanfile_v2.format(name=name,
                                                                    version=version,
                                                                    package_name=package_name),
-             "test_package/example.cpp": test_main.format(name=name),
-             "test_package/CMakeLists.txt": test_cmake_v2.format(name=name)}
+             "test_package/src/example.cpp": test_main.format(name=name),
+             "test_package/src/CMakeLists.txt": test_cmake_v2.format(name=name)}
+    return files
+
+
+conanfile_exe = """from conans import ConanFile
```

Comment:
Nonrelated question, are we moving/copying the ConanFile to `conan` namespace at some point?

Reply:
Yes, we are.
I didn't want to do it yet, because I am thinking of using the move to clean some potentially breaking things, and the opt-in would be the new import.